src/app/View.ts

```diff
@@ -118,7 +129,25 @@ export class View {
     const containerGame = Render.elementFactory('div', {
       className: 'container-game',
     });
-    //render elements
+
+    rollADiceButton.addEventListener('click', () => {
+      const rollResult = gameController.breed();
+      herdView.innerText = JSON.stringify(
+        newPlayer.theHerd.theAnimals.map(([animal, count]) => [
+          animal.theName,
+          count,
+        ]),
+      );
+      rollResultView.innerText = JSON.stringify(rollResult);
+      gameController.nextPlayer();
+      gameController.startTurn();
+    });
+    // TODO find a solution for the Timer - now it's not working right
+    setInterval(() => {
+      remainingTime.innerText = `Time Left : ${newTimer.theTurnTimeLeft} s`;
+    }, 1000);
```

Comment:
Infinite interval. Fix this.

Create method to display remaining time in view, and move getting timer to controller